db/document.go

```diff
@@ -316,7 +363,7 @@ func (doc *document) getNonWinningRevisionBody(revid string, loader RevLoaderFun
 func (doc *document) getRevisionBodyJSON(revid string, loader RevLoaderFunc) []byte {
 	var bodyJSON []byte
 	if revid == doc.CurrentRev {
-		bodyJSON, _ = json.Marshal(doc.body)
+		bodyJSON, _ = json.Marshal(doc._body)
```

Comment:
The error is being ignored here.  I think this should at least log a warning if it's painful to propagate the error to callers.